keps/sig-network/784-kube-proxy-component-config/README.md

```diff
@@ -127,6 +135,55 @@ The mitigations to those risks:
 
 ## Design Details
 
+### Following sections will be added
+| Field   | Comments                                            |
+|---------|-----------------------------------------------------|
+| Linux   | new section for linux (platform-specific) options   |
+| Windows | new section for windows (platform-specific) options |
```

Comment:
I think the plan is to get rid of windows directly, is not likely anybody is going to work on it

Reply:
Well, we talked about this in SIG Network this week.

At any rate, having all the Windows-specific config in its own section(s) could be a step toward moving winkernel out of tree, if that eventually becomes the plan.

Reply:
I thought we were going to do like with userspace proxy, just remove it completely

Reply:
Someone has to maintain windows service proxying somewhere.

At any rate, my point was that if we eventually want the windows proxy code to be not-in-tree (whether that's because it's somewhere else, or because it no longer exists), then having all the windows-specific config all in one place wil lmake it easier to delete it later.